test/index.html

```diff
@@ -56,15 +56,14 @@
                 }
 
                 /* global BoxAnnotations */
-                var annotations = new BoxAnnotations(null, { features: { discoverability: true, drawing: true } });
+                var annotations = new BoxAnnotations(null, { features: {} });
 
                 /* global Box */
                 var preview = new Box.Preview();
 
                 preview.show(fileid, token, {
                     container: '.preview-container',
                     boxAnnotations: annotations,
-                    enableAnnotationsDiscoverability: true,
                     enableAnnotationsImageDiscoverability: true,
```

Comment:
Should we remove the image discoverability feature, as well?

Reply:
I spoke with @ConradJChan , seems like a good idea, will update!